modules/nodes-scheduler-node-affinity-configuring-preferred.adoc

```diff
@@ -19,32 +19,43 @@ The following steps demonstrate a simple configuration that creates a node and a
 $ oc label node node1 e2e-az-name=e2e-az3
 ----
 
-. In the `Pod` spec, use the `nodeAffinity` stanza to configure the `preferredDuringSchedulingIgnoredDuringExecution` parameter:
+. Create a pod with a specific label:
 +
-.. Specify a weight for the node, as a number 1-100. The node with highest weight is preferred.
+.. Create a YAML file with the following content:
 +
-.. Specify the key and values that must be met. If you want the new pod to be scheduled on the node you edited, use the same `key` and `value` parameters as the label in the node:
+[NOTE]
+====
+You cannot add an affinity directly to an existing scheduled pod.
+====
 +
 [source,yaml]
 ----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: s1
 spec:
-  affinity:
+  affinity: <1>
     nodeAffinity:
-      preferredDuringSchedulingIgnoredDuringExecution:
-      - weight: 1
+      preferredDuringSchedulingIgnoredDuringExecution: <2>
+      - weight: <3>
         preference:
           matchExpressions:
-          - key: e2e-az-name
-            operator: In
+          - key: e2e-az-name <4>
             values:
             - e2e-az3
+            operator: In <5>
+#...
 ----
-+
-.. Specify an `operator`. The operator can be `In`, `NotIn`, `Exists`, `DoesNotExist`, `Lt`, or `Gt`. For example, use the Operator `In` to require the label to be in the node.
+<1> Adds a pod affinity.
+<2> Configures the `preferredDuringSchedulingIgnoredDuringExecution` parameter.
+<3> Specifies a weight for the node, as a number 1-100. The node with highest weight is preferred.
+<4> Specifies the `key` and `value` that must be met. If you want the new pod to be scheduled on the node you edited, use the same `key` and `value` parameters as the label in the node.
```

Comment:
In a couple of other files in this PR, I see similar text that mentions "the `key` and `values` that must be met". Is `value` correct here, or should it be `values`?